### ISSUE TITLE
Review Shoulds and Considers from VBA RO Pilot MVP after CMS collab cycle touchpoint

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#15989]
## Description or Additional Context 

CMS collab cycle feedback was documented in ticket #16073.

#### MUST

- [x] All character counters MUST behave the same across all screens of this product ~~and MUST align with other character counters in the CMS.~~ Collaboration between a designer on this team and the CMS team may be necessary to determine the best path forward on how character counters are handled in this product. This is handled separately in #16523. 
- [x] Separate ticket to address: Error messages MUST be determined for all input fields prior to product launch. Use clientside validation in as many places as it makes sense to display errors prior to form submit (CMS Team still working on this). This is handled separately in #16068.

#### SHOULD

- [x] All [Forms, Details Components](https://prod.cms.va.gov/admin/appearance/styleguide#form-details) SHOULD all be left aligned and full width, unless there is reason for them not to be.
- [x] Radio and checkbox inputs SHOULD be unselected by default, unless there is reason for them not to be.
- [x] All input fields SHOULD have meaningful help text that use plain language. Several fields in the VA Service Taxonomy page currently do not have help text or have help text that uses technical jargon.
- [x] ~~Service Region name input field on the VA Service Regions page should have help text. Is there any standardized way that these should be named? If so, describe that.~~

#### CONSIDER

- [x] For fields that are hidden depending on if certain conditions are met (for example, "Use the facility street address" and "Use the facility phone number" checkboxes, CONSIDER showing both of the states in your design for fields that show depending on when the box is checked vs unchecked.
- [x] CONSIDER whether [Forms, Details Components](https://prod.cms.va.gov/admin/appearance/styleguide#form-details) in the service location paragraph type will be open or closed by default.
- [x] For sections where users will be able to add multiple contact methods in the service location paragraph type, CONSIDER how users will be able to easily distinguish between the contact methods in the UI when multiple are added.
- [x] CONSIDER whether free text inputs on the VA Service Taxonomy page would be the most appropriate for this type of content.

**Notes:** 

- @davidmpickett to confirm character counts for all text input fields

## Design
[VBA Content Specs Figma file](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=208%3A714&mode=design&t=IePMjbSbOBYTX5xg-1)

## Acceptance Criteria
- [x] Review SHOULD and CONSIDER items. 
  - [x] Note in comments which items have been implemented.
  - [x] ~~Stub tickets for items that may be candidates for later.~~ No new tickets needed to be made.
- [x] Request UX Lead review (usually 48 hours turnaround)

**Comment by xiongjaneg** [comment]
From Nov. 3 scrum: Showing taxonomy info impacts a limited number of people, mostly regional and national level editors, so it's out of scope for Pilot MVP.

**Comment by davidmpickett** [comment]
Adding collab cycle ticket as blocker just for findability

**Comment by davidmpickett** [comment]
Some of my notes from the call today:

- Rich-text editor will be changing with Drupal 10 upgrade, but there's not an updated component in the design system yet
- Character counts are currently handled in a couple different ways - this will be a good opportunity for Jordan and Blake to work on which should be the standard approach
- Client-side validation will be new functionality coming to Drupal soon. We should look to see if any of these fields would benefit from using that immediate type of feedback
- Designs are all focused on Happy path right now, what about error states?

**Comment by davidmpickett** [comment]
Just added a task for me to review character count on all text fields.

**Comment by davidmpickett** [comment]
In sprint planning today we discussed this as potentially blocked by #16068 since this is a catchall ticket and that's a specific set of work. 

We also discussed that the character count feedback might be good to split off into it's own ticket since it will require collaboration with CMS Platform team.

**Comment by xiongjaneg** [comment]
Stubbed character count ticket #16345

**Comment by xiongjaneg** [comment]
@jilladams @mmiddaugh Per discussion in UX sync, pulling this ticket out of 99 pending some discussion with CMS Team on some collab cycle feedback.

**Comment by jilladams** [comment]
Based on planning, pulling out of Sprint 100. Jordan may have capacity to start this work as stretch, but likely won't get far enough to request review.

**Comment by jilladams** [comment]
@xiongjaneg pls break off the specific update of the VBA character counter work for Jordan to do.

**Comment by xiongjaneg** [comment]
@thejordanwood #16523 stubbed character count for your refinement, thank you!

**Comment by thejordanwood** [comment]
SHOULD feedback update:

- All [Forms, Details Components](https://prod.cms.va.gov/admin/appearance/styleguide#form-details) are left aligned and full width when possible. If not left aligned or full width, they mirror the current styling in the CMS.
-  Radio and checkbox inputs are unselected by default, unless there is reason for them not to be. In the Service location paragraph type, several of these are selected by default  to encourage editors to use facility data when possible. This also prevents more fields appearing on the page and reduces the clutter. 
-  All input fields now have meaningful help text. Help text has been added to the fields in the VA Service Taxonomy.

**Comment by thejordanwood** [comment]
CONSIDER feedback update:

- All fields that were hidden depending on certain conditions are now shown. 
- All accordions in the service location paragraph type will be closed by default. I believe my mockup was a bit confusing because it shows all accordions open to help engineers understand the full design. I now show the default state as closed. 
-  On the service location paragraph type, there is already a design that helps distinguish between contact methods. I now show that in the Figma file. 
-  On the VA Service Taxonomy page, there may have been some confusion about what the free text input fields are for. New help texts now provides a better explanation and I think this field type is appropriate.

**Comment by jilladams** [comment]
@thejordanwood implication there is this is ready for @aklausmeier feedback when she's back to office?

**Comment by thejordanwood** [comment]
@jilladams Yes, this is ready for her review.

**Comment by thejordanwood** [comment]
@jilladams For reporting purposes, this ticket can be considered closed. All ACs are complete, but I'd like to leave this ticket open to capture any feedback Amanda may have. 

cc: @xiongjaneg

**Comment by jilladams** [comment]
Removed from sprint 102 since we believe it's done, but if more work comes in from feedback, feel free to slap that back on here.

**Comment by aklausmeier** [comment]
Reviewed, looks good. IMO this ticket can close. @thejordanwood cc @jilladams
